store/migrations/migration0/migrate.go

```diff
@@ -0,0 +1,294 @@
+package migration0
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/big"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/araddon/dateparse"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/smartcontractkit/chainlink/store/models"
+	"github.com/smartcontractkit/chainlink/store/orm"
+	"github.com/tidwall/gjson"
+	null "gopkg.in/guregu/null.v3"
+)
+
+type Migration struct{}
+
+func (m Migration) Timestamp() string {
+	return "0"
+}
+
+func (m Migration) Migrate(orm *orm.ORM) error {
+	orm.InitializeModel(&JobSpec{})
+	orm.InitializeModel(&JobRun{})
+	orm.InitializeModel(&Initiator{})
+	orm.InitializeModel(&Tx{})
+	orm.InitializeModel(&TxAttempt{})
+	orm.InitializeModel(&BridgeType{})
+	orm.InitializeModel(&IndexableBlockNumber{})
+	orm.InitializeModel(&User{})
+	orm.InitializeModel(&Session{})
+	orm.InitializeModel(&ServiceAgreement{})
+	return nil
+}
+
+type Time struct {
+	time.Time
+}
+
+// UnmarshalJSON parses the raw time stored in JSON-encoded
+// data and stores it to the Time field.
+func (t *Time) UnmarshalJSON(b []byte) error {
+	var n json.Number
+	if err := json.Unmarshal(b, &n); err != nil {
+		return err
+	}
+	newTime, err := dateparse.ParseAny(n.String())
+	t.Time = newTime.UTC()
+	return err
+}
+
+type JobSpec struct {
+	ID        string `json:"id" storm:"id,unique"`
+	CreatedAt Time   `json:"createdAt" storm:"index"`
+	JobSpecRequest
+}
+
+// JobSpecRequest represents a schema for the incoming job spec request as used by the API.
+type JobSpecRequest struct {
+	Initiators []Initiator `json:"initiators"`
+	Tasks      []TaskSpec  `json:"tasks" storm:"inline"`
+	StartAt    null.Time   `json:"startAt" storm:"index"`
+	EndAt      null.Time   `json:"endAt" storm:"index"`
+}
+
+type RunStatus string
+
+type RunResult struct {
+	JobRunID     string      `json:"jobRunId"`
+	Data         models.JSON `json:"data"`
+	Status       RunStatus   `json:"status"`
+	ErrorMessage null.String `json:"error"`
+	Amount       *big.Int    `json:"amount,omitempty"`
+}
+
+type TaskType string
+
+type TaskSpec struct {
+	Type          TaskType    `json:"type" storm:"index"`
+	Confirmations uint64      `json:"confirmations"`
+	Params        models.JSON `json:"-"`
+}
+
+// UnmarshalJSON parses the given input and updates the TaskSpec.
+func (t *TaskSpec) UnmarshalJSON(input []byte) error {
```

Comment:
Have we got a database which would trigger this code? I'm not sure it's actually doing anything.